packages/core-assortments/src/module/configureAssortmentMediaModule.ts

```diff
@@ -75,7 +75,12 @@ export const configureAssortmentMediaModule = async ({
       },
     );
 
-    return AssortmentMediaTexts.findOne(selector, {});
+    const mediaTexts = await AssortmentMediaTexts.findOne(selector, {});
```

Comment:
That is not the way:
1. Extra fetch per locales, no good
3. When updating multiple locales for ex. 3 locales, it will add 3 extra fetches + emit the UPDATE TEXT event 3 times instead of once